cmd/main.go

```diff
@@ -15,16 +14,13 @@ import (
 )
 
 func main() {
-	// 1. Load config
```

Comment:
I don't think we should delete comments like this even though we think that it is unnecessary and obvious